src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -216,6 +217,22 @@ class BinaryCompactObject : public DomainCreator<3> {
     static type default_value() { return true; }
   };
 
+  struct UseLogarithmicMapOuterSphericalShell {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Use a logarithmically spaced radial grid in the outer spherical "
+        "shell."};
+    static type default_value() noexcept { return false; }
+  };
+
+  struct OuterSphericalShellLayers {
+    using type = size_t;
+    static constexpr OptionString help = {
+        "The number of radial layers in the outer sphere."};
```

Comment:
I wondering whether it would be beneficial to split the current Layer 4 into Layers 4 and 5, where the new Layer 4 would refer exclusively to the 10 Frustum blocks, and Layer 5 refers to the Wedge3D blocks enveloping them. That way this help string can be changed to `The number of layers in Layer 5, the outer sphere.`